src/components/Tab.ts

```diff
@@ -0,0 +1,61 @@
+import { div, h } from 'react-hyperscript-helpers';
+import { Clickable } from 'src/components/common';
+import { icon } from 'src/components/icons';
+import colors from 'src/libs/colors';
+import * as Style from 'src/libs/style';
+
+const styles = {
+  container: {
+    display: 'flex',
+    alignItems: 'center',
+    fontWeight: 500,
+    textTransform: 'uppercase',
+    marginTop: '1rem',
+    height: '3.75rem',
+  },
+  button: (isActive: boolean) => ({
```

Comment:
When reading code that calls this function, there's no hint what the flag means (`styles.button(true)` or `styles.button(false)`, so you have to refer back here.

Suggest using an options object to make this clearer.

```suggestion
  button: ({ isActive = false }: { isActive?: boolean } = {}) => ({
```

It's much easier to understand `styles.button({ isActive: true })` vs `styles.button(true)`.

https://alexkondov.com/should-you-pass-boolean-to-functions/